lib/api/core/entryPoints/index.js

```diff
@@ -15,6 +16,7 @@ EntryPoints.prototype.init = function entryPointsInit () {
   return this.http.init()
     .then(() => {
       this.proxy.init();
+      return Promise.resolve();
```

Comment:
Potential conflict with https://github.com/kuzzleio/kuzzle/pull/541 since there is no HTTP server anymore